runtests.ps1

```diff
@@ -0,0 +1,7 @@
+# Run the basic sets of tests using powershell

```

Comment:
This file shouldn't be top-level - can you move this into a directory within Tools?